### PR TITLE
Add Tray icon

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -27,19 +39,98 @@ const createWindow = (): void => {
...
+    };
+  }
+
+  // TASKBAR BOTTOM
+  // Since all the other cases were handled, we can be sure we're dealing with a bottom taskbar
+  return {
+    x: Math.round(trayBounds.x + trayBounds.width / 2 - WIDTH / 2),
+    y: workArea.height - HEIGHT,
+  };
+};
```
Comment (GuillaumeRx) [inline src/main.ts:98]: could this be a switch ?

Comment (FrederikBolding) [reply src/main.ts:98]: Hmm, how would you suggest? @GuillaumeRx

**Comment by GuillaumeRx** [comment]
Capture d’écran 2020-09-28 à 12 24 41
On mac it's a bit weird

File: src/main.ts (hunk 1 above)
Comment (GuillaumeRx) [reply src/main.ts:98]: After some testing it can't be a switch, good as it is now

File: src/main.ts [hunk 2]
```diff
@@ -27,19 +39,98 @@ const createWindow = (): void => {
...
+  window.show();
+  window.focus();
+};
+
+const toggleWindow = () => {
+  if (window.isVisible()) {
+    window.hide();
+  } else {
+    showWindow();
+  }
```
Comment (GuillaumeRx) [inline src/main.ts:114]: you can use ternary operators here

File: src/main.ts [hunk 3]
```diff
@@ -27,19 +39,98 @@ const createWindow = (): void => {
...
+};
+
+const createTray = () => {
+  tray = new Tray(path.join(__dirname, 'favicon.png'));
```
Comment (GuillaumeRx) [inline src/main.ts:118]: `favicon.png` should be a 32x32px so it renders well in macos

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 2 above)
Comment (FrederikBolding) [reply src/main.ts:114]: I don't understand how you would use ternaries here, can you explain?

**Comment by FrederikBolding** [comment]
I think the new favicon could fix the MacOS issue as well, please check 👍

File: src/main.ts (hunk 2 above)
Comment (GuillaumeRx) [reply src/main.ts:114]:
```ts
window.isVisible() ? window.hide() : showWindow()
```

**Comment by GuillaumeRx** [comment]
Capture d’écran 2020-09-28 à 17 16 17
Still a bit big 🤔

**Comment by FrederikBolding** [comment]
Added a 16x16 version for Mac OS @GuillaumeRx

**Review by GuillaumeRx — APPROVED** [review]
Good now 👌

**Pull request merged**